types/stompjs/index.d.ts

```diff
@@ -56,8 +56,8 @@ export class Frame {
 
     toString(): string;
     sizeOfUTF8(s: string): number;
```

Comment:
According to https://github.com/jmesnil/stomp-websocket/blob/master/src/stomp.coffee#L51 `static` should be added here too.